cli/tests/integrations/test_task.py

```diff
@@ -132,11 +130,11 @@ def test_log_pod_task():
             ['dcos', 'task', 'log', 'good-container', 'stderr'])
 
         # pod task log are not executor logs, so normal executor stderr
-        # logs shouldn't be seen and this pod shoudn't have any logging
+        # logs shouldn't be seen and this pod shouldn't have any logging
         # to stderr
-        assert returncode == 1
-        assert stderr == b'No files exist. Exiting.\n'
-        assert stdout == b''
+        assert returncode == 0
+        assert stderr == b''
```

Comment:
stdout doesn't print a newline if it succeeds. It's probably doing a newline print of the the stderr file. If the file is empty, we should probably tell the user as much.